app_tests/integration_tests/llm/sglang/conftest.py

```diff
@@ -0,0 +1,123 @@
+# Copyright 2024 Advanced Micro Devices, Inc.
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import json
+import logging
+import os
+import pytest
+
+from ..utils import (
+    find_available_port,
+    start_llm_server,
+    download_with_hf_datasets,
+    export_paged_llm_v1,
+    compile_model,
+)
+
+pytest.importorskip("sglang")
+import sglang as sgl
```

Comment:
typically i see this written as

sgl = pytest.importorskip("sglang")

However, I think this looks nicer because although there are concerns with repeat import, import is supposed to be idempotent anyways & this way we get a syntax highlighted "import" keyword.

Reply:
Yeah, I had a double thought about this, but decided to keep it. You don't get any IntelliSense in the editor if you do `sgl = pytest.importorskip("sgl")`. Worth it to import it in line below, to make writing the tests much easier.